examples/gwlb_with_vmseries/variables.tf

```diff
@@ -264,7 +264,7 @@ variable "load_balancers" {
 
     Please refer to [module documentation](../../modules/loadbalancer/README.md#frontend_ips) for available properties.
 
-    > [!NOTE] 
+    > [!NOTE]
```

Comment:
This admonition will not show properly, use this instead (like in `panorama` code for example):
`**Note!** \`

Reply:
Fixed.